github3/github.py

```diff
@@ -996,7 +996,8 @@ def pubsubhubbub(self, mode, topic, callback, secret=''):
         :returns: bool
         """
         from re import match
-        m = match('https?://[\w\d\-\.\:]+/\w+/[\w\._-]+/events/\w+', topic)
+        m = match('https?://[\w\d\-\.\:]+/\w+[\w-]+\w/[\w\._-]+/events/\w+',
```

Comment:
Based on what you said GitHub's sign-up allows, shouldn't the username regexp be `\w[\w-]+\w` because then you'd be able to have `a-b` or `a-bcdefg-h` etc.?